tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -406,3 +406,52 @@ def test_rh_cloud_tag_values(
                 if tag['key'] == 'host_collection':
                     assert tag['value'] == f'"{host_col_name}"'
                     break
+
+
+@pytest.mark.run_in_one_thread
+@pytest.mark.tier2
+def test_positive_tag_values_max_length(
+    inventory_settings,
+    organization_ak_setup,
+    rhcloud_registered_hosts,
+    rhcloud_sat_host,
+    default_sat,
```

Comment:
Please rebase your branch on the current master branch and change `default_sat` to `target_sat` as there is no `default_sat` fixture anymore.
See @JacobCallahan's email sent out on May 20, 2022, to satqe-list.

Reply:
updated!